prime-router/settings/STLTs/OH/oh-doh.yml

```diff
@@ -0,0 +1,200 @@
+- name: "oh-doh"
+  description: "Ohio Department of Health"
+  jurisdiction: "STATE"
+  stateCode: "OH"
+  countyName: null
+  filters:
+    - topic: "covid-19"
+      jurisdictionalFilter: []
+      qualityFilter: null
+      routingFilter:
+        - "matches(abnormal_flag, A)"
```

Comment:
Need to add as following FHIRPATH for the matches(abnormal_flag, A) to conditionFilter instead here:

- "(%resource.interpretation.coding.code = 'A')"

Reply:
Done! Good catch!